server/command.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+const COMMAND_HELP = `* |/zoom start| - Start a zoom meeting.`
+
+func getCommand() *model.Command {
+	return &model.Command{
+		Trigger:          "zoom",
+		DisplayName:      "Zoom",
+		Description:      "Integration with Zoom.",
+		AutoComplete:     true,
+		AutoCompleteDesc: "Available commands: start",
+		AutoCompleteHint: "[command]",
+	}
+}
+
+func (p *Plugin) postCommandResponse(args *model.CommandArgs, text string) {
+	post := &model.Post{
+		UserId:    p.botUserID,
+		ChannelId: args.ChannelId,
+		Message:   text,
+	}
+	_ = p.API.SendEphemeralPost(args.UserId, post)
+}
+
+func (p *Plugin) executeCommand(c *plugin.Context, args *model.CommandArgs) (string, error) {
+	config := p.getConfiguration()
+
+	split := strings.Fields(args.Command)
+	command := split[0]
+	action := ""
+
+	if command != "/zoom" {
+		return fmt.Sprintf("Command '%s' is not /zoom. Please try again.", command), nil
+	}
+
+	if len(split) > 1 {
+		action = split[1]
+	} else {
+		p.postCommandResponse(args, "Please specify an action for /zoom command.")
```

Comment:
You don't need this line any longer, because https://github.com/mattermost/mattermost-plugin-zoom/pull/36/files#diff-7278f3371df5ef589578d375bde66febR133 will post the message.

Reply:
oh right, my mind was else where. THanks